buildSrc/src/main/kotlin/otel.errorprone-conventions.gradle.kts

```diff
@@ -25,6 +25,8 @@ tasks {
         disable("SystemOut")
       }
 
+      disable("BooleanParameter")
```

Comment:
How about putting this at bottom to apply to test / jmh?

Reply:
Do we actually want this check in our main code? I find it useless.

Reply:
Will let others suggest too - I think it helps in at least some cases, but it's not one I'd push too hard for.